doc/users/interactive.rst

```diff
@@ -280,25 +270,24 @@ If ``ipympl`` is installed use the magic:
 
 to select and enable it.
 
-If you only need to use the classic notebook you can use
+If you only need to use the classic notebook, you can use
 
 .. sourcecode:: ipython
 
   %matplotlib notebook
```

Comment:
Side note, did we at some point decide to use `%matplotlib nbagg`? I thought we decided one was better than the other (though they're aliases).

Reply:
it is called `nbagg` on our side which should read as "N B Agg', but it was pointed out that if you are not familiar with our backend naming conventions the obvious the obvious reading is "N Bag" which is not great, hence the `%matplotlib notebook` alias.  I think we should prefer `notebook` when writing docs.